### DISCUSSION TITLE
Bug: pick/omit labels appear backwards in the interactive rebase levels

### DISCUSSION THREAD

**Discussion by annahansen229** [pcottle/learnGitBranching#1029] [Ideas]
In the interactive rebase levels, the omit/pick labels on the buttons are swapped from what they should be. 

I think the intention may have been to label the action that happens when you click the button - i.e., click this to omit it/click this to pick it. But since you are re-creating the appearance of the interactive rebase text editor, it creates confusion because you would expect it to say "pick" next to a commit that you want to keep. 

Perhaps a more clear approach to satisfy both labeling the action of the button and accurately representing an interactive rebase text editing session would be to include the 'pick'/'omit' text to the left of the commit ID, then group the drag/drop control and the button together with some visual separation from the text.

![image](https://user-images.githubusercontent.com/33067494/196465557-47f6ca0f-87c1-421e-9c6a-48f72a67b2ba.png)

**Comment by pcottle** [comment]
Yeah thats a good point! Would you like to attempt a fix by submitting a PR? It should be a somewhat easy diff, just a re-order and a logic swap

**Reply by alex4cable** [reply]
I also got confused and assumed to have a bug here. Finalizing the exercise was possible when understanding that if you want to 
pick the button has to be omit
and if you want to 
omit the button has to be pick.

Definitely a point to improve to avoid confusion.
